flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -614,6 +614,67 @@ public void testParameterizedTypes() throws Exception {
 		executor.closeSession(sessionId);
 	}
 
+	@Test
+	public void testCreateTable() throws Exception {
+		final Executor executor = createDefaultExecutor(clusterClient);
+		final SessionContext session = new SessionContext("test-session", new Environment());
+		String sessionId = executor.openSession(session);
+		try {
+			executor.useCatalog(sessionId, "catalog1");
```

Comment:
We also need tests:
1. create table with full qualified name `create table catalog1.db1.table1 (...) with (...)`
2. create table with only database name and table name `create table db1.table1 (...) with (...)`
3. create table with computed column
4. create table with watermark definition